lib/components/InputField/InputField.tsx

```diff
@@ -0,0 +1,38 @@
+import { useId } from 'react';
+
+export interface InputFieldProps {
+  value: string;
+  onChange: (newValue: string) => void;
+  placeholder?: string;
+  label?: string;
```

Comment:
Label should be always present for the field as the screenreader user does not know what the field is for. Especially if some text is already in it.

If it is a must to have visually hidden then the aria-label should still be available for screenreader.

Reply:
I change the label from optional to mandatory.